ks/vojtux_common.ks

```diff
@@ -204,9 +207,6 @@ EOM
 chmod 755 /usr/local/bin/monitor-toggle
 echo "Preparing accessibility override..."
 cat > /etc/dconf/db/local.d/01-accessibility <<- EOM
-# Be nice to the users and pre-enable screen reading if they decide to install Gnome.
```

Comment:
This dconf section is not related to the sound theme, please keep it in place. It should go into vojtux-settings package which you introduce in a separate PR.

Reply:
Removed from spec file, added back to common kickstart.